package.json

```diff
@@ -154,6 +154,7 @@
 		"whybundled:server": "whybundled client/stats-server.json",
 		"composite-checkout-demo": "webpack-dev-server --config ./packages/composite-checkout/webpack.config.demo.js --mode development",
 		"components:storybook:start": "start-storybook -c packages/components/.storybook",
+		"language-picker:storybook:start": "start-storybook -c packages/language-picker/.storybook",
```

Comment:
when I run this, I get an error on the storybook page in the browser "Exports is read only"

Reply:
I'm not sure why this suddenly isn't working 🤔 Reading the error below:

```./packages/language-picker/node_modules/@automattic/react-i18n/node_modules/@wordpress/is-shallow-equal/lib/objects.js/<@http://localhost:50609/vendors~main.74bd29d19c2e02c9b421.bundle.js:213500:1
```

It appears to be an issue within a dependency. The storybook worked before so I'm wondering if at some point I changed something while working on it that caused it to break (like updating the dependencies 🤔)

Reply:
FWIW it works well for me now.